### ISSUE TITLE
GIVbacks harvest modal is giving bad math

### ISSUE THREAD

**Issue by GriffGreen** [Giveth/giveth-dapps-v2#788]
<img width="782" alt="Screen Shot 2022-05-20 at 10 23 21 AM" src="https://user-images.githubusercontent.com/14945613/169571125-6e87840d-3788-45ea-831f-9b76e7c5be26.png">

CS did well... but not that well... It seems like the amount was doubled in this modal.. here is how much they really got:

https://blockscout.com/poa/xdai/tx/0x859a3c1b6d67bc5a7958b05debea495a4314120c958c1522cdc15a709d4de46a

**Comment by MohammadPCh** [comment]
@laurenluz , Yes I fixed it in #799 . I agree with you. we can reduce givback stream amount from it.

**Comment by divine-comedian** [comment]
this doesn't look great either and might still confuse people 
![image](https://user-images.githubusercontent.com/67759413/173627050-e51f2712-0473-4de5-bd06-9ec23e6a4e16.png)


new idea @MohammadPCh 

let's just subtract the givback stream from the reward stream, what do you think? tried it out on my local and it looks good:
![image](https://user-images.githubusercontent.com/67759413/173627785-ab86b0b5-f8c8-427f-bd48-fbd1da9f3c68.png)

let me know and I'll make a PR for the changes for you to merge

**Comment by MohammadPCh** [comment]
@divine-comedian I agree with you. But if we want to subtract the GIVback stream from the reward stream, why don't we show the GIVbacks row? I think @markoprljic  @mosaeedi can help us better :)

**Comment by divine-comedian** [comment]
> @divine-comedian I agree with you. But if we want to subtract the GIVback stream from the reward stream, why don't we show the GIVbacks row? hey, @markoprljic @mosaeedi we need your help :D

I'm not sure. I have a feeling there was a complicated reason why it was designed like that in the first place, I think this fix is good enough without making it too complicated.

**Comment by mosaeedi** [comment]
I think it's good for now, but it still looks confusing and we have to make it simpler later, but for now, we can live with this. Thanks @divine-comedian @MohammadPCh

**Comment by divine-comedian** [comment]
okay here is the PR @MohammadPCh 

https://github.com/Giveth/giveth-dapps-v2/pull/968
